Core/include/Acts/Geometry/ConeVolumeBounds.hpp

```diff
@@ -0,0 +1,228 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/Volume.hpp"
+#include "Acts/Geometry/VolumeBounds.hpp"
+#include "Acts/Utilities/BoundingBox.hpp"
+#include "Acts/Utilities/Definitions.hpp"
+
+#include <cmath>
```

Comment:
System includes should probably come first.

I am not sure if we already have guidelines for include ordering. I think clang-format might also do the ordering automatically.

Reply:
Oh, I see - I thought is was the other way round, but makes sense, I guess.

Reply:
Can we clarify this - and then I make PRs for the Core packages ?

Reply:
Yes, I think that would be good. There actually is an option for clang-format to sort this automatically.